src/server/api/lib/twilio.js

```diff
@@ -403,12 +429,36 @@ async function handleIncomingMessage(message) {
   }
 }
 
+/**
+ * Create a new Twilio messaging service
+ */
+async function createMessagingService(organization, friendlyName) {
+  const twilio = await getTwilio(organization);
+  return await twilio.messaging.services.create({
+    friendlyName,
+    statusCallback: process.env.TWILIO_STATUS_CALLBACK_URL,
+    inboundRequestUrl: process.env.TWILIO_INBOUND_URL
```

Comment:
Use a base url instead, e.g. TWILIO_BASE_CALLBACK_URL instead of the two different env vars. You also need to add the organization id to the `inboundRequestUrl` to get validation to work with TWILIO_MULTI_ORG

See:
https://github.com/MoveOnOrg/Spoke/blob/main/src/server/api/lib/twilio.js#L26-L43